PL2/PL2-shared/src/test/java/nl/tudelft/pl2016gr2/model/MetaDataTest.java

```diff
@@ -0,0 +1,42 @@
+package nl.tudelft.pl2016gr2.model;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class MetaDataTest {
+  
+  private MetaData md;
```

Comment:
I do not agree with this one, in general I'm not using long variable names and metaData is much longer than md.